lib/signal-handler.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var QEmitter = require('qemitter'),
+    signalHandler = new QEmitter();
+
+module.exports = signalHandler;
+
+process.on('SIGHUP', notifyAndExit(1));
+process.on('SIGINT', notifyAndExit(2));
```

Comment:
Thats a realy good idea, and I was thinking about it, but decided, that it is not related to this PR.